kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -444,6 +444,18 @@ module.controller('grImageMetadataCtrl', [
       ctrl.metadata = metadata;
       ctrl.usageRights.first().data = usageRights;
     };
+
+    ctrl.isDomainMetadataEmpty = (key) => {
+      return ctrl.domainMetadata.find(obj => obj.name === key ).fields.every(field => field.value === undefined );
+    };
+
+    ctrl.isAdditionalMetadataEmpty = () => {
+      var totalAdditionalMetadataCount = Object.keys(ctrl.metadata).filter(key=> { ctrl.isUsefulMetadata(key); }).length +
```

Comment:
```suggestion
      const totalAdditionalMetadataCount = Object.keys(ctrl.metadata).filter(key => ctrl.isUsefulMetadata(key)).length +
```

I think you'll need to take the braces off the anonymous function here in order to correctly filter on the response